### ISSUE TITLE
YOLO26 openvino export in older ultralytics faster compared to the newest version, when inferencing with iGPU.

### ISSUE THREAD

**Issue by kgbvimpel4** [ultralytics/ultralytics#23288]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

We are noticing a performance regression when exporting the YOLO26 model to OpenVINO using newer versions of Ultralytics. Specifically, inference time on iGPU has slowed down significantly, whereas CPU performance remains consistent. Interestingly, this issue seems specific to this model, as our tests with YOLOv11n and YOLOv8n did not show similar degradation.

Please find the benchmark results, the export CLI command, and the testing code attached below for review.

### Environment

**yolo export model=yolo26n.pt format=openvino half=true**

Ultralytics 8.4.4 🚀 Python-3.12.3 torch-2.9.1+cu128
openvino version: 2025.4.1-20426-82bbf0292c5-releases/2025/4
Model: yolov26n new
----------------------------------------
Warming up (10 runs)...
Benchmarking (100 runs)...
  Average: 40.91 ms
  Min:     39.26 ms
  Max:     57.56 ms
  FPS:     24.45

Ultralytics 8.3.129  Python-3.10.13 torch-2.3.1+cu121
openvino version: 2024.2.0-15519-5c0f38f83f6-releases/2024/2
Model: yolov26n old
----------------------------------------
Warming up (10 runs)...
Benchmarking (100 runs)...
  Average: 20.39 ms
  Min:     19.32 ms
  Max:     37.52 ms
  FPS:     49.05

### Minimal Reproducible Example

[benchmark.py](https://github.com/user-attachments/files/24677242/benchmark.py)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @kgbvimpel4, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

Thanks for the detailed benchmarks and for sharing your script 🙏 — performance regressions (especially iGPU-specific) can be tricky, and we’ll want to reproduce this on our side. This is an automated response 🤖, and an Ultralytics engineer will also assist soon.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. You already attached `benchmark.py` ✅—to make it fully reproducible, please also include:
-Exact `ultralytics` version(s) tested (you provided these ✅)
-Exact export + inference commands used end-to-end (including any OpenVINO device selection and batch/imgsz settings)
-Hardware details (iGPU model) and OS
-If possible, the smallest public model + input that still shows the slowdown (even a single image), or precise steps we can follow to reproduce with `yolo26n.pt`

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
Exporting YOLO26 with a version lower than 8.4.0 would simply result in a broken model. You can run prediction and see that the predictions are wrong. YOLO26 requires 8.4.0 and higher.

**Comment by kgbvimpel4** [comment]
> Exporting YOLO26 with a version lower than 8.4.0 would simply result in a broken model. You can run prediction and see that the predictions are wrong. YOLO26 requires 8.4.0 and higher.

Yes, we tested the pretrained yolo26n.pt model, which failed. However, after fine-tuning the model with our custom dataset and testing it with OpenVINO, resulted in successfull output. Notably, the model exported with the older version was actually more accurate than the newer version.

**Comment by Y-T-G** [comment]
It's probably because the older version doesn't use NMS. You would probably get the same accuracy increase by removing the NMS free head after training with 8.4 which would make it NMS based like YOLO11

```python
from ultralytics import YOLO

model = YOLO("best.pt")
del model.model.model[-1].one2one_cv2
```

NMS based model has better accuracy but slower postprocessing speed. NMS free model has lower accuracy, but faster postprocessing. Your benchmark script doesn't include postprocessing time. Although it could also be the case the ops introduced in NMS free model is not accelerated when running with OpenVINO GPU which would make it slower than NMS based models.

**Comment by brmarkus** [comment]
Are benchmark results shared regularly along with release notes?
I'm curious how the performance of Yolo26 (and more, future variants) "develops" over time.
Where could such benchmark results be found?

**Comment by glenn-jocher** [comment]
We don’t currently publish a per-release, continuously updated benchmark table in the release notes, since perf depends heavily on hardware/drivers and runtime settings; for the most comparable “over time” view, we recommend running the same `yolo benchmark` command on your target machine each time you upgrade and tracking the results alongside the [GitHub releases](https://github.com/ultralytics/ultralytics/releases) and the docs pages like [Benchmark mode](https://docs.ultralytics.com/modes/benchmark/) and the [OpenVINO integration guide](https://docs.ultralytics.com/integrations/openvino/).

```bash
yolo benchmark model=yolo26n.pt data=coco8.yaml
```
